src/models/Models.ts

```diff
@@ -207,7 +207,7 @@ export interface RequestWithAuth extends Request {
 export interface IPlatform extends Document {
   name: string;
   credits: number;
-  whitelist: string[];
+  whitelist: Map<string, Map<string, string[]>>;
```

Comment:
Add a comment above that explains it, e.g.
`// assetType -> chainId -> address[]`